distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFanout.java

```diff
@@ -14,31 +14,48 @@
 package ddf.test.itests.catalog;
```

Comment:
For all the tests you've updated here, the coverage numbers haven't changed at all?